package.json

```diff
@@ -312,6 +324,7 @@
   },
   "resolutions": {
     "**/lodash": "4.17.15",
-    "**/js-yaml": "^3.13.1"
+    "**/js-yaml": "^3.13.1",
+    "cypress": "^4.5.0"
```

Comment:
Dumb question: Why do we need to use selective dependency resolution here?

Full disclosure: My knowledge of [selective dependency resolution](https://classic.yarnpkg.com/en/docs/selective-version-resolutions/) is limited, and the [RFC](https://github.com/yarnpkg/rfcs/blob/master/implemented/0000-selective-versions-resolutions.md) was only minimally helpful.

Reply:
Nope, not a dumb question. CircleCI was getting itself confused and failing without this. I think one of the Cypress dependencies conflicted with the main Cypress install or something like that.

I'm thinking of upgrading to 4.8.0 (or later if available) soon after rolling out. I can try removing this when I do that to see if it's still necessary.